libexec/tfenv-install

```diff
@@ -159,6 +159,15 @@ download_signature() {
     || log 'error' 'SHA256SUMS signature download failed';
 };
 
+# If on MacOS with Homebrew, use GNU grep
+# This allows keybase login detection to work on Mac
+if [[ $(uname) == 'Darwin' ]] && [ $(which brew) ]; then
```

Comment:
Perhaps, this would be better re-written as:
```
if [[ $(uname) == 'Darwin' ]] && [[ command -v brew ]]; then
```

Notes:
- consistent use of `[[` test construct rather than `[`
- uses `command -v` built-in rather than `which` to find the command